coinbase_tests.py

```diff
@@ -150,10 +150,62 @@ def tearDown(self):
 
         self.page.driver.quit()
 
+class HomePageSetup(unittest.TestCase):
 
+    def setUp(self):
+        self.driver = webdriver.Chrome(root_path + config.win_chromedriver_path)
+        self.page = HomePage(self.driver)
+    
+    def tearDown(self):
+
+        self.page.driver.quit()
+
+
+class HomePageCustomerSecurity(HomePageSetup):
+    """
+    Confirm that the four links concerning the safety and security of the
+    customer's funds are functional.
+
+    Acceptance Criteria
+    -------------------
+    -Clicking on the Customer fund are secure link navigates the user to the 
+     corresponding url.
+    -Clicking on the Insurance service link navigates the user to the 
+     corresponding url.
+    -Clicking on the Adherence of idustry practice link navigates the user to
+     the corresponding url.
+    -Clicking on the Wallet security link navigates the user to the 
+     corresponding url.  
+    """
+
+    def testSecurityLinks(self):
+        driver = self.driver
+        page = self.page
 
+        page.secure_storage_link.click()
+        self.assertEqual(page.driver.title,'Secure Bitcoin Storage - Coinbase')
+        self.assertEqual(
+            page.driver.current_url,'https://www.coinbase.com/security')
+        driver.back()
 
```

Comment:
I debated using driver.back() or creating three more test methods. Let me know which is better.